configuration/multimedia.md

```diff
@@ -9,7 +9,7 @@ title: Multimedia
 
 ## Volume
 
-The framework supports some base [functions](https://www.openhab.org/javadoc/v2.5/org/eclipse/smarthome/model/script/actions/audio#method.summary) to control the audio sinks' volume.
+The framework supports some base [functions](https://next.openhab.org/javadoc/latest/org/openhab/core/model/script/actions/audio#method.summary) to control the audio sinks' volume.
```

Comment:
Do we really want to use `next.openhab.org`?
I would assume that we will move to our usual domain, after the release.

@ghys and @kaikreuzer any thoughts on this?

Reply:
Those links needs to be updated in other pages as well:

```bash
$ grep -R next.openhab.org
README.md:The result is available at [https://next.openhab.org/docs/](https://www.openhab.org/docs/) and [https://next.openhab.org/addons/](https://www.openhab.org/addons/).
README.md:on most of the documentation pages on https://next.openhab.org/, 
.vuepress/docs-sidebar.js:        path: 'https://next.openhab.org/javadoc/latest/'

```

Reply:
We can probably do that with a little script afterwards then anyways so it would be ok for me to have the next openhab docs working well for now.

Reply:
Yes let's merge it! :slightly_smiling_face: